docs/guides/upgrading-to-v1.0.md

```diff
@@ -15,7 +15,7 @@ v0.14, run:
 
 ```console
 # Update to the latest CRDs.
-kubectl apply -f https://raw.githubusercontent.com/crossplane/crossplane/release-1.0/cluster/charts/crossplane/crds
+kubectl apply -k https://github.com/crossplane/crossplane//cluster?ref=release-1.0
```

Comment:
Is this actually what we want to apply? The idea was to only apply the CRDs.

Reply:
@negz this should only apply crds https://github.com/crossplane/crossplane/blob/4e38db255343f423a21b90af2615d5fb55eb45bc/cluster/kustomization.yaml#L3

Reply:
Oh, missed the `-k`. Nice.